<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-YOURSY" ignored="0" name="listener test" tests="2" failures="1" timestamp="14 一月 2018 02:39:46 GMT" time="0.031" errors="0">
  <testcase name="testfun1" time="0.0" classname="com.tt.learning.TestListenerDemo"/>
  <testcase name="testfun2" time="0.015" classname="com.tt.learning.TestListenerDemo">
    <failure type="java.lang.AssertionError" message="expected [2] but found [1]">
      <![CDATA[java.lang.AssertionError: expected [2] but found [1]at com.tt.learning.TestListenerDemo.testfun2(TestListenerDemo.java:17)... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- testfun2 -->
</testsuite> <!-- listener test -->
